viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -196,9 +196,10 @@ export class Cognite3DViewer {
   /**
    * Reusable buffers used by functions in Cognite3dViewer to avoid allocations.
    */
-  private readonly _updateNearAndFarPlaneBuffers = {
-    combinedBbox: new THREE.Box3(),
-    bbox: new THREE.Box3()
+  private readonly _boundingBoxes = {
+    nearFarPlaneBoundingbox: new THREE.Box3(),
+    innerBoundingbox: new THREE.Box3(),
```

Comment:
I'm not sure `innerBoundingbox` is a clear name, but not sure about what it should be either :thinking: Could it be `baseBoundingBox` or `modelBoundingBox`?

Also, we usually write `Box` in `boundingBox` with a capital `B`

Reply:
Done